css/main.css

```diff
@@ -325,3 +325,61 @@ li {
 .skills {
   list-style: inside url("../media/svg/circle.svg");
 }
+
+.contact-me {
+  display: flex;
+  flex-direction: column;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+}
+
+.social-media ul {
+  display: flex;
+  fill: rgb(34, 33, 33);
+  justify-content: center;
+}
+
+.contact-text {
+  margin-right: 15%;
+  margin-left: 15%;
+  margin-top: 20px;
+  margin-bottom: 20px;
+  text-align: center;
+  font-size: 18px;
+}
+
+input {
+  border-top: 0;
+  border-right: 0;
+  border-left: 0;
+  margin-left: 5%;
+  margin-right: 5%;
+  padding-bottom: 10px;
+}
+
+textarea {
+  border-top: 0;
+  border-right: 0;
+  border-left: 0;
+  border-bottom: 0;
+  margin-left: 5%;
+  margin-right: 5%;
+}
+
+#footer-button {
+  margin-left: 35%;
+  margin-right: 30%;
+  background-color: #ff6b00;
+  border: none;
+  color: white;
+  padding: 10px;
+  margin-top: 40px;
+  margin-bottom: 40px;
+}
+
+.email-container {
+  text-align: center;
+}
```

Comment:
- [ ]  Please stick to the design templates in Figma as much as possible

![image](https://user-images.githubusercontent.com/95243996/190298056-598d24ac-1316-412d-825d-fd1fd972d811.png)

- [ ]  please note that the color of the bottom border of the inputs is lighter than the color you used, also note that the placeholders contain padding.

- [ ]  Please note that the contact me section on the figma design takes the whole viewport height of the sceen.

- [ ] Please note that the text area contains a background color and is higher than the one you have in yout project 

- [ ]  Kyndly use the correct fonts and font sizes to stick as much as you can to the figma design.